examples/variational_inference/bayesian_neural_network_advi.myst.md

```diff
@@ -199,17 +203,38 @@ plt.xlabel("iteration");
 trace = approx.sample(draws=5000)
 ```
 
-Now that we trained our model, lets predict on the hold-out set using a posterior predictive check (PPC). We can use {func}`~pymc.sample_posterior_predictive` to generate new data (in this case class predictions) from the posterior (sampled from the variational estimation).
+Now that we trained our model, lets predict on the hold-out set using a posterior predictive check (PPC). We can use {func}`pymc.sample_posterior_predictive` to generate new data (in this case class predictions) from the posterior (sampled from the variational estimation).
+
+To make this work with our mutable {class}`Data` objects, we need to create a new model object that removes the minibatches, and predicts on the whole data set. Notice that we are using our fitted `trace` to sample from the posterior predictive distribution, using the posterior estimates from the original model. The {class}`Flat` distribution is just a placeholder to make the model work; the actual values are sampled from the posterior.
```

Comment:
> To make this work with our mutable {class}`Data` objects,

This justification doesn't make sense. Instead say, to predict on a whole dataset ...